registration_ref/app.py

```diff
@@ -2,27 +2,37 @@
 from time import sleep
 from typing import Optional
 
-from flask import Flask, abort, jsonify, request
+from flask import Flask, abort, jsonify, request, Response
 import requests
+import logging
 
 from registration_ref.crypto import sign_device_csr
 from registration_ref.sota_toml import sota_toml_fmt
 from registration_ref.settings import Settings
 
 app = Flask(__name__)
 
+logging.basicConfig(level=logging.DEBUG)
+gunicorn_logger = logging.getLogger('gunicorn.access')
+app.logger.handlers = gunicorn_logger.handlers
+app.logger.setLevel(gunicorn_logger.level)
 
 @app.before_request
 def _auth_user():
     pass
 
+@app.after_request
+def _log_request(res: Response):
+    app.logger.info(f"{request.remote_addr} - {request.method} {request.path} - {res.status_code}")
+    return res
 
 def log_device(uuid: str, pubkey: str):
+    app.logger.debug(f"Logging device with UUID: {uuid}, pubkey: {pubkey}")
```

Comment:
Just my general nit that nobody else seems to agree with: If log.debug is enabled - you still force python the create this fairly large string. If you do the original log style of `log.debug("logging device with UUID: %s pubkey: %s", uuid, pubkey)` then python won't actually build the string unless log.debug is enabled.

Reply:
Interesting, so if you plug in enough f strings into debug messages it can have an impact even if you do not run Debug mode.

Reply:
Addressed as well